completions/dpkg

```diff
@@ -13,20 +13,12 @@ if _comp_have_command grep-status; then
 else
     _comp_xfunc_dpkg_installed_packages()
     {
-        command grep -A 1 "Package: ${1-}" /var/lib/dpkg/status 2>/dev/null |
-            command grep -B 1 -Ee "ok installed|half-installed|unpacked| \
-            half-configured" \
-                -Ee "^Essential: yes" |
-            awk "/Package: ${1-}/ { print \$2 }" 2>/dev/null
+	command awk -F'\n' '-vRS=\n\n' "\$1 ~ /Package: $1/ && (\$2 ~ /ok installed|half-installed|unpacked|half-configured/ || \$2 ~ /^Essential: yes/) {print(substr(\$1, 10)); }" /var/lib/dpkg/status 2>/dev/null
```

Comment:
Thank you for your contribution. I generally agree with the idea of making it a single invocation of an external command, but `RS` with more than one character is unspecified by POSIX.